### DISCUSSION TITLE
Where is the Auth API documentation?

### DISCUSSION THREAD

**Discussion by KevinLourd-Belong** [supabase/supabase#19069] [Questions] [answered]
**Where is the Auth Documentation for the Auth API endpoints?**

I am building a no-code website (with Bubble.io) that uses Supabase for Auth.

I would need some API endpoints for signup, login, send magic link, reset password, etc.

I could find `/auth/v1/signup` by chance, but couldn't find the doc. For login, I tried `/auth/v1/token` but failed to make it work

**Comment by GaryAustin1** [comment, marked as answer]
https://github.com/supabase/gotrue#endpoints

**Reply by KevinLourd-Belong** [reply]
Thank you, beautiful
